css-three-column-layout/styles.css

```diff
@@ -0,0 +1,69 @@
+body {
+  background-color: #1996da;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
```

Comment:
You should break up this `row` into multiple utility classes too that you could reuse. I would name the something like `justify-center` and `align-center` and then give them their specific property each.